dio/lib/src/interceptor.dart

```diff
@@ -214,13 +214,13 @@ typedef InterceptorSendCallback = void Function(
 
 /// The signature of [Interceptor.onResponse].
 typedef InterceptorSuccessCallback = void Function(
-  Response<dynamic> e,
+  Response<dynamic> response,
   ResponseInterceptorHandler handler,
 );
 
 /// The signature of [Interceptor.onError].
 typedef InterceptorErrorCallback = void Function(
-  DioException e,
+  DioException err,
```

Comment:
I'd prefer `error`.

Reply:
Changing `e` to `err` is done to maintain consistency with the existing [Interceptor.onError](https://github.com/cfug/dio/blob/a7a787840b87def9d16c10c009c548ddfcd86337/dio/lib/src/interceptor.dart#L202) method. If `err` in [Interceptor.onError](https://github.com/cfug/dio/blob/a7a787840b87def9d16c10c009c548ddfcd86337/dio/lib/src/interceptor.dart#L202) were changed to `error`, it would cause complaints from downstream packages' CI or IDE.

```dart
/// Called when an exception was occurred during the request.
void onError(
  DioException err,
  ErrorInterceptorHandler handler,
) {
  handler.next(err);
}
```

Reply:
Thanks for the explanation! In this case I think it's better to be `error` and we can maybe consider update the callback in the next breaking version.

Reply:
That's great. I'm going to try changing all instances of `err` to `error`. If it looks good, I can recommit the commit based on the 6.0.0 branch.